examples/ChatQnA/deploy/aipc.md

```diff
@@ -0,0 +1,757 @@
+# Single node on-prem deployment with Ollama on Xeon Scalable processors
+
+This deployment section covers single-node on-prem deployment of the ChatQnA
```

Comment:
@mkbhanda @dbkinder I was adhering to the Xeon template and didn't change the common sections such as this. I understand that this will apply to many other places in the APIC document. I can apply these changes forming a new template for AIPC if it's okay with you.

Reply:
Yes, we don't want to change the "common" sections at this point.  Right @preethivenkatesh @tomlenth